tests/epyccel/test_epyccel_sign.py

```diff
@@ -0,0 +1,185 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring/
+
+import numpy as np
+import modules.numpy_sign as mod
+
+from pyccel.epyccel import epyccel
+
+def test_sign_complex(language):
+    f_pos = mod.complex_pos
+    f_neg = mod.complex_neg
+    f_pos_epyc = epyccel(f_pos, language = language)
+    f_neg_epyc = epyccel(f_neg, language = language)
+
+    assert f_pos_epyc() == f_pos()
+    assert f_neg_epyc() == f_neg()
```

Comment:
Please add the requested type checks to this file

Reply:
https://github.com/pyccel/pyccel/pull/1169/commits/ba06f25b8b31da097dad9a55ff2a75e1eaf93940